### PR TITLE
Dz3

### PR DESCRIPTION
**Pull request by Oleksii-Romanchenko**

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:wght@400;500;700;900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./css/styles.css">
+</head>
+  <body>
+    <header class="header">
+      <div class="conteiner-header">
```
Comment (Oleksii-Romanchenko) [inline portfolio.html]: typo in the word contAiner

Comment (romanchenko31) [reply portfolio.html]: зробив замину

File: css/pr.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <style>
+         
+.wrapper{
+    display:flex;
+    width: 500px;
+    background-color: rgb(255, 255, 255);
+    border: 1px solid black;
+    
+}
+.cont{
```
Comment (Oleksii-Romanchenko) [inline css/pr.html:19]: cont ?

File: css/pr.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    width: 200px;
+    height: 200px;
+    background-color: blueviolet;
+}
+.conter{
```
Comment (Oleksii-Romanchenko) [inline css/pr.html:24]: conter ? find out detter description

File: css/pr.html (hunk 2 above)
Comment (romanchenko31) [reply css/pr.html:19]: pr.html this is file for trening

File: css/pr.html (hunk 3 above)
Comment (romanchenko31) [reply css/pr.html:24]: pr.html this is file for trening

**Pull request merged**